Remove references to InfluxDB Aware as we no longer offer it. (#5005)
influxdata/docs-v2: content/enterprise_influxdb/v1.10/administration/monitor/_index.md

- [Use InfluxQL for diagnostics](/enterprise_influxdb/v1.10/administration/monitor/diagnostics/)

{{% note %}}
### Monitor with InfluxDB Insights

> For InfluxDB Enterprise customers, Insights is a free services that can monitor your cluster. InfluxDB Insights sends monitoring metrics for your cluster to a private Cloud account. This allows the support team to monitor your cluster health as well as making resource usage statistics available to assist with support tickets that you raise.

Reviewer: ```suggestion
For InfluxDB Enterprise customers, Insights is a free service that monitors your cluster and sends metrics to a private Cloud account. This allows InfluxDB Support to monitor your cluster health and access usage statistics when assisting with support tickets that you raise.
```
